src/conf.c

```diff
@@ -468,6 +468,57 @@ int ConfGetChildValueIntWithDefault(const ConfNode *base, const ConfNode *dflt,
     return ret;
 }
 
+/**
+ * \brief Retrieve the value of a child configuration node as string.
+ *
+ * This function will return the value for a configuration node based
+ * on the full name of the node. In case the node exists but does not
```

Comment:
In this case we're not using the full name of the node. We're just using the name relative to the `base` node being passed in.  Something better might be:
```
This function retrieves the value of a configuration node by name that exists under the provided base node.
```